### ISSUE TITLE
Users: Use Turbolinks to update user info

### ISSUE THREAD

**Issue by maxdobeck** [maxdobeck/zeiter#21]
Page shouldn't require a full reload after updating user info.  Should reload only the elements in question.

**Comment by kapoorlakshya** [comment]
Is this something we do in the HTML (components) or is this server side?

**Comment by maxdobeck** [comment]
Both, if you're defining an HTML component as the show.html.erb page.  Gem is already installed just have to follow these steps:
1. (Done) Add the turbolinks gem, version 5, to your Gemfile: gem 'turbolinks', '~> 5.0.0'
2. (Done) Run bundle install.
3. Add //= require turbolinks to your JavaScript manifest file (usually found at app/assets/javascripts/application.js).

Unofficially step 4 is to add a jQuery event listener:
```
document.addEventListener("turbolinks:load", function() {
  // ...
})
```
Do NOT use `document.onReady`.  This will basically break everything.  You have to use `turbolinks:load` or other turbolinks api.

Then follow this [GoRails](https://gorails.com/episodes/upgrade-to-turbolinks-5?autoplay=1) video to see the full upgrade + use process for Turbolinks 5.

Additional Resources:
* [Quick 3 minute example](https://www.youtube.com/watch?v=3Eo4dlg7Y0c)

**Comment by kapoorlakshya** [comment]
Ah, got it. So we have to use JS for it.

**Comment by maxdobeck** [comment]
Yeah jQuery technically does all the heavy lifting via ajax requests
